packages/open-next/src/adapters/cache.ts

```diff
@@ -122,21 +144,17 @@ export default class S3Cache {
           kindHint?: "app" | "pages" | "fetch";
           tags?: string[];
           softTags?: string[];
+          kind?: "FETCH";
         },
   ) {
     if (globalThis.disableIncrementalCache) {
       return null;
     }
-    const isFetchCache =
-      typeof options === "object"
-        ? options.kindHint
-          ? options.kindHint === "fetch"
-          : options.fetchCache
-        : options;
+    const _isFetchCache = isFetchCache(options);
 
     const softTags = typeof options === "object" ? options.softTags : [];
     const tags = typeof options === "object" ? options.tags : [];
-    return isFetchCache
+    return _isFetchCache
```

Comment:
```suggestion
    return isFetchCache(options)
```

and remove `_isFetchCache`